README.md

```diff
@@ -17,4 +17,106 @@ doctrine:
                 mappings:
                     AddressingBundle:
                         is_bundle: true
+```
+
+You'll also need to add some javascript code, to make sure the form changes on
+changing the country code work.
+
+The script below gives an example. entity_name is the snake_case formation of
+the entity that contains the address field and has the address form on it's
+form.
+
+```javascript
```

Comment:
Maybe mention this is only the case if you work with symfony4/yarn else the require statement won't work